angular-portfolio/.gitignore

```diff
@@ -0,0 +1,49 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
+
+# compiled output
+/dist
+/tmp
+/out-tsc
+# Only exists if Bazel was run
+/bazel-out
+
+# dependencies
+/node_modules
+
+# profiling files
+chrome-profiler-events*.json
+speed-measure-plugin*.json
+
+# IDEs and editors
+/.idea
+.project
+.classpath
+.c9/
+*.launch
+.settings/
+*.sublime-workspace
+
+# IDE - VSCode
+.vscode/*
+!.vscode/settings.json
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
+.history/*
+
+# misc
+/.sass-cache
+/connect.lock
+/coverage
+/libpeerconnection.log
+npm-debug.log
+yarn-error.log
+testem.log
+/typings
+
+# System Files
+.DS_Store
+Thumbs.db
+
+# Other Files
+/home/billyporter/step/angular-portfolio/backend/src/main/webapp/style.css
```

Comment:
"home/billyporter" is outside the git repo. It also uses relative paths, so you can leave out step and angular-portfolio. Finally, since we don't want any compiled files here, you can use an asterisk to indicate all files of type css, js, and html.

So I think this should be:

/backend/src/main/webapp/*.js
/backend/src/main/webapp/*.css
/backend/src/main/webapp/*.html

And remember that to have the file removed from the repo, you will also have to run the ""git rm --cached FILENAME" command from the command line.